ngx-fudis/projects/ngx-fudis/src/lib/directives/grid/grid-utils.spec.ts

```diff
@@ -28,6 +31,13 @@ describe('GridUtils', () => {
 
       expect(correctReturnString).toBe('1/-1');
 
+      fudisGridAlignItemsArray.forEach((value) => {
```

Comment:
Since all of these test functions are lengthy, I would organise them to follow the same order as presented in ts file. Etc, put 'getGrid Classes' as the first test, improves readability.